3.3/examples/django/run-in-app-container.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Use this script to run one-off commands inside a container of a frontend pod
+# (where you application code lives in)
+#
+# Examples:
+# ./run-in-app-container.sh date
+# ./run-in-app-container.sh env
+# ./run-in-app-container.sh ./manage.py migrate
+
+quoted_args="$(printf " %q" "$@")"
+osc exec -p $(osc get pods -l name=frontend -t '{{ with index .items 0 }}{{ .metadata.name }}{{ end }}') -it -- bash -c "cd \$HOME && scl enable python33 \"$quoted_args\""
```

Comment:
@soltysh it did NOT work without explicitly doing `scl enable python33`. I'd rather have it work without it as well. Somehow, without that:

```bash
$ osc exec -p $(osc get pods -l name=frontend -t '{{ with index .items 0 }}{{ .metadata.name }}{{ end }}') -it -- bash -c "python --version"
Python 2.7.5
```